modules/rosa-rn-security.adoc

```diff
@@ -0,0 +1,55 @@
+:_module-type: CONCEPT
+// Module included in the following assemblies:
+//
+// rosa_release_notes/rosa-rn-new-features.adoc
+
+[id="rosa-rn-security_{context}"]
+= Security
+
+[id="rosa-rn-auth-provider_{context}"]
+== Authentication provider
+
+Authentication for the cluster can be configured using either the link:https://cloud.redhat.com/openshift[OpenShift Cluster Manager (OCM)] cluster creation process or using the `rosa` CLI.
+
+OpenShift Container Platform is not an identity provider, and all access to the cluster must be managed by customers as part of their integrated solution. {product-title} (ROSA) supports the use of multiple identity providers that are simultaneously provisioned.
+
+The following identity providers are supported:
+
+* GitHub or GitHub Enterprise
+* GitLab
+* Google
+* LDAP
+* OpenID connect
+
+[id="rosa-rn-priv-containers_{context}"]
+== Privileged containers
+
+Privileged containers are supported on ROSA with the `cluster-admin` role. To enable Red Hat to operate ROSA as a managed service with an service level agreement (SLA), some restrictions are enforced to limit the ability of rogue or accidental changes that could impact the service.
```

Comment:
This is _also_ no longer true, and has been updated in https://www.openshift.com/products/dedicated/service-definition#privileged-containers

Reply:
Fixed. Need to add a link to the Service Definition when it is published.